frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_2__Cell Type and Gene Ordering.mdx

```diff
@@ -4,7 +4,7 @@ Cell types and genes shown in a dot plot can be arranged in different ways as de
 
 # Cell Type Ordering
```

Comment:
This link needs to be updated to match the pinned ontology in the schema?

### Algorithm

`1. Load the CL ontology, using either the cl.obo or cl.owl from the source file [here](https://github.com/obophenotype/cell-ontology/releases/tag/v2021-08-10).`